fetches/ssr/operationUnits/OperationUnits.jsx

```diff
@@ -7,7 +7,9 @@ async function getOperationUnits() {
     fetch(
       `${reqApiHost(
         req,
-      )}/api/v2/operation-units?expand=openingHours,town.county,parameters`,
+      )}/api/v2/operation-units?expand=${
+        process.env.NEXT_PUBLIC_OPERATION_UNITS_EXPAND
+      }`,
```

Comment:
ale pozor, ak neexistuje process.env.NEXT_PUBLIC_OPERATION_UNITS_EXPAND v nejakom projekte (mobilonline?) tak sa moze stat, ze to prestane posielat expand. Preto to sprav, tak ako to mame na viacerych miestach, ze ak existuje tak pouzi a ak neexistuje tak tam daj ten defaul co tam bol

Reply:
@ivankopcik zapracované